Dockerfile

```diff
@@ -100,7 +100,7 @@ RUN \
 ## As per https://docs.docker.com/engine/reference/builder/#scope, ARG need to be repeated for each scope
 ARG JENKINS_AGENT_VERSION=4.13-1-alpine-jdk11
 
-LABEL io.jenkins-infra.tools="helm,kubectl,helmfile,sops,aws-cli,aws-iam-authenticator,yamllint,updatecli,jenkins-agent,doctl,jq,yq,ruby"
+LABEL io.jenkins-infra.tools="aws-cli,aws-iam-authenticator,azure-cli,doctl,helm,helmfile,jenkins-agent,jq,kubectl,ruby,sops,updatecli,yamllint,yq"
```

Comment:
Sorting these values now means sorting them every time. OK for you?

Reply:
yeah I don't mind :)

Reply:
sorted is easier to read / find what you are after :)